packages/client/components/GitHubScopingSearchFilterMenu.tsx

```diff
@@ -75,109 +67,90 @@ type GitHubSearchQuery = NonNullable<
   NonNullable<GitHubScopingSearchFilterMenu_viewer['meeting']>['githubSearchQuery']
 >
 
-const getValue = (item: {name: string}) => item.name.toLowerCase()
-
-const MAX_PROJECTS = 10
+const setSearch = (atmosphere: Atmosphere, meetingId: string, value: string) => {
+  commitLocalUpdate(atmosphere, (store) => {
+    const meeting = store.get(meetingId)
+    if (!meeting) return
+    const githubSearchQuery = meeting.getLinkedRecord('githubSearchQuery')!
+    githubSearchQuery.setValue(value, 'reposQuery')
+  })
+}
 
 const GitHubScopingSearchFilterMenu = (props: Props) => {
-  // TODO replace projects
   const {menuProps, viewer} = props
   const isLoading = viewer === null
-  const projects = viewer?.teamMember?.integrations.github?.projects ?? []
+  const atmosphere = useAtmosphere()
+  const edges = viewer?.teamMember?.integrations?.github?.api?.query?.search.edges ?? []
   const meeting = viewer?.meeting ?? null
   const meetingId = meeting?.id ?? ''
   const githubSearchQuery = meeting?.githubSearchQuery ?? null
   const nameWithOwnerFilters = githubSearchQuery?.nameWithOwnerFilters ?? []
-  const {fields, onChange} = useForm({
-    search: {
-      getDefault: () => ''
-    }
-  })
-  const {search} = fields
-  const {value} = search
-  const query = value.toLowerCase()
-  const showSearch = projects.length > MAX_PROJECTS
-  const queryFilteredProjects = useFilteredItems(query, projects, getValue)
-  const selectedAndFilteredProjects = useMemo(() => {
-    const selectedProjects = projects.filter((project) => nameWithOwnerFilters.includes(project.id))
-    const adjustedMax =
-      selectedProjects.length >= MAX_PROJECTS ? selectedProjects.length + 1 : MAX_PROJECTS
-    return Array.from(new Set([...selectedProjects, ...queryFilteredProjects])).slice(
-      0,
-      adjustedMax
-    )
-  }, [queryFilteredProjects])
+  const reposQuery = githubSearchQuery?.reposQuery ?? ''
 
-  const atmosphere = useAtmosphere()
-  const {portalStatus, isDropdown} = menuProps
-  const toggleJQL = () => {
-    commitLocalUpdate(atmosphere, (store) => {
-      const searchQueryId = SearchQueryId.join('github', meetingId)
-      const githubSearchQuery = store.get(searchQueryId)
-      // this might bork if the checkbox is ticked before the full query loads
-      if (!githubSearchQuery) return
-      githubSearchQuery.setValue([], 'nameWithOwnerFilters')
-    })
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Could clear the search input here when the menu closes but a user that has searched, selected and closed the menu, may wish to quickly open it again and deselect the issue. Having to search again to deselect would be a pain. Alternatively, we could clear the search but show all selected repos at the top of the list

Reply:
yeaaaah showing the selected things at the top gets my vote! is that what we do for atlassian? i can't remember